src/subtables/mod.rs

```diff
@@ -135,8 +135,8 @@ where
     &self,
     dense: &DensifiedRepresentation<F, C>,
     r_mem_check: &(F, F),
-  ) -> [GrandProducts<F>; S::NUM_MEMORIES] {
-    std::array::from_fn(|i| {
+  ) -> Vec<GrandProducts<F>> {
```

Comment:
Can you toggle this on the "multicore" feature?

Reply:
just put a PR (#51) implementing this

Reply:
Thank you @0xfuturistic appreciate the tag team man!